src/ucp/core/ucp_request.inl

```diff
@@ -826,4 +890,18 @@ ucp_request_invoke_uct_completion_success(ucp_request_t *req)
     return UCS_OK;
 }
 
+
+#define UCP_SEND_REQUEST_GET_BY_ID(_req_p, _worker, _req_id, _extract, \
```

Comment:
remove "send" from all macros/function names related to the request id

Reply:
done